File: Selenium/Activities/Activity10.py
from selenium import webdriver
from selenium.webdriver.common.by import By

with webdriver.Chrome() as driver:

    driver.get("https://training-support.net/webelements/drag-drop")

    print("Title of the page " + driver.title)

    actions = webdriver.ActionChains(driver)
    football = driver.find_element(By.XPATH, "//*[@alt = 'A Football']")
    dropzone1 = driver.find_element(By.XPATH, "//*[@id = 'dropzone1']")
    dropzone2 = driver.find_element(By.XPATH, "//*[@id = 'dropzone2']")
    dropZone1Text = driver.find_element(By.XPATH, "//*[text() = 'Dropzone1']").text
    dropZone2Text = driver.find_element(By.XPATH, "//*[text() = 'Dropzone2']").text

    actions.click_and_hold(football).move_to_element(dropzone1).release().pause(2).perform()
    assert dropZone1Text !=dropzone1
    print("Dropzone1 is changed to " + dropzone1.text+ " Ball is dropped in Dropzone1")

    actions.drag_and_drop(football, dropzone2).pause(2).perform()
    assert dropZone2Text != dropzone2
    print("Dropzone2 is changed to " + dropzone2.text + " Ball is dropped in Dropzone2")

    driver.close()

File: Selenium/Activities/Activity10.java

import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.WebElement;
import org.openqa.selenium.chrome.ChromeDriver;
import org.openqa.selenium.interactions.Actions;

public class Activity10{
	
	public static void main(String[] args) {
		
		WebDriver driver = new ChromeDriver();
		driver.get("https://training-support.net/webelements/drag-drop");
		
		Actions builder = new Actions(driver);
		
		WebElement football = driver.findElement(By.xpath("//*[@alt = 'A Football']"));
		WebElement Dropzone1 = driver.findElement(By.xpath("//*[@id = 'dropzone1']"));
		WebElement Dropzone2 = driver.findElement(By.xpath("//*[@id = 'dropzone2']"));
		String DropZone1Text = driver.findElement(By.xpath("//*[text() = 'Dropzone1']")).getText();		
		String DropZone2Text = driver.findElement(By.xpath("//*[text() = 'Dropzone2']")).getText();
		
		builder.clickAndHold(football).moveToElement(Dropzone1).release().pause(2000).build().perform();
		
//		builder.dragAndDrop(football, Dropzone1).pause(2000).build().perform();
		
		assert DropZone1Text != "Dropzone1";
		System.out.println("Dropzone1 is changed to " + Dropzone1.getText() + " Ball is dropped in Dropzone1");
		
		builder.dragAndDrop(football, Dropzone2).pause(2000).build().perform();
		assert DropZone2Text != "Dropzone2";
		System.out.println("Dropzone2 is changed to " + Dropzone2.getText() + " Ball is dropped in Dropzone2");
		
		driver.close();
	}
}